conf/env.sh.example

```diff
@@ -31,3 +31,8 @@ ITERATIONS=3
 SLEEP=30
 # Compact levels with less than the following possible nodes after loads
 COMPACT_CUTOFF=$((256**3 + 1))
+# To run map reduce jobs a shaded jar is built, the following properties
+# determine what versions of Fluo and Accumulo client libs end up in the shaded
+# jar.
+FLUO_VERSION=$($FLUO_HOME/bin/fluo version)
+ACCUMULO_VERSION=$(accumulo version)
```

Comment:
Should mention in README.md that accumulo command should be on PATH. It already mentions that yarn and hadoop should be there.